src/lib.rs

```diff
@@ -91,12 +70,15 @@ pub trait Message: Send + 'static {
 /// }
 /// ```
 #[async_trait::async_trait]
-pub trait Handler<M: Message>: Actor {
+pub trait Handler<M>: Actor {
+    /// The return value of this handler.
+    type Return: Send + 'static;
```

Comment:
Should this be kept as Result to keep it with the old naming convention?

Reply:
Sure, I don't mind :)

I figured `Return` is slightly more appropriate because `Result` in Rust usually refers to the enum and we can have `Handler`s that do _not_ return a `Result` but just `()` or are otherwise infallible.

I don't feel strongly about that though!

Reply:
Fair enough. I borrowed Result from Actix I think. Likewise though, I don't feel very strongly about it, so we can keep it as Return as far as I'm concerned

Reply:
If I get the choice I'll stick with `Return` :)

Reply:
I think that makes sense